iam/hasher.py

```diff
@@ -0,0 +1,30 @@
+import base64
+import hashlib
+import hmac
+import secrets
+
+
+def new_salt(n=12):
+    salt_chars = ('ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz'
```

Comment:
You could use

    import string
    ....
    string.ascii_letters + string.digits
    # 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789'

I'm not saying it's better, perhaps a bit visually more appealing.